docs/wiki/basic-tutorials/running.md

```diff
@@ -0,0 +1,191 @@
+---
+keywords:
+- run bot
+- running bot
+- execute bot
+- deployment
+- application
+---
+
+
+# Running and Deploying your Bot
+
+If you took the time to write a bot, at some point you'll also want to run it, either for productive use or to debug it from the IDE.
+
+## :construction_worker: Running from your IDE
+
+While developing your bot, you will want to run your bot directly from the IDE in order to quickly test changes and new features. For this, create a Run/Debug Configuration in your IDE of choice with your bots main class. Remember to also add any necessary parameters and environment variables.
+
+A working Run/Debug configuration will also enable you to run your bot with a debugger. A debugger is often considered a developers most important tool, so make sure to familiarize yourself with the debugging integration for your IDE of choice.
+
+### IntelliJ IDEA
+
+This assumes your project is set up correctly, preferably with [Gradle](/wiki/getting-started/intellij-gradle), that it can be built without errors, and that it does not yet have any run/debug configurations.
+
+**1.** Locate and click the `Add Configuration...` button at the top bar next to the start button.
+
+![](./running-idea-configurations-empty.png)
+
+**2.** Another window opens. Click the `+` button in the top left and select `Application`
```

Comment:
The wording is a bit awkward here, maybe change this to be consistent with line 51 and stay in an imperative voice? Something like:
```suggestion
**2.** In the newly opened window, click the `+` button in the top left and select `Application`
```